rc.go

```diff
@@ -157,6 +161,16 @@ func (rc *RC) Load(ctx context.Context, config *Config, env Env) (newEnv Env, er
 		direnv,
 		rc.Path(),
 	)
+
+	// Allow RC loads to be canceled with SIGINT
+	ctx, cancel := context.WithCancel(context.Background())
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt)
```

Comment:
When `signal.Notify` is called, it changes the runtime. Once the function is exited, the runtime is still configured to send signals to that channel.

I think the original design was a bit cleaner in that regard since both were decoupled. Not a big issue in practice since rc.Load is always called once.